multi/stm32l4-multi/tty.c

```diff
@@ -309,12 +310,22 @@ static void tty_irqthread(void *arg)
 		}
 		mutexUnlock(ctx->irqlock);
```

Comment:
On imx driver I got such comment:

"imho this check should be performed under mutex"

Maybe it applies also here?

Reply:
In this case we need to synchronize with `tty_setCflag()` - but it is not protected by any lock. So you're right, but this is non-trivial change

Reply:
so I will merge it as is, synchronization should be done as an independent change.